devsetup/Makefile

```diff
@@ -102,38 +93,33 @@ crc_attach_default_interface_cleanup: ## Detach default libvirt network from CRC
 ##@ EDPM
 
 .PHONY: edpm_baremetal
-edpm_baremetal: export EDPM_BMH_NAMESPACE=${BMH_NAMESPACE}
-edpm_baremetal: export EDPM_PROVISIONING_INTERFACE=${BM_PROVISIONING_INTERFACE}
-edpm_baremetal: export EDPM_CTLPLANE_INTERFACE=${BM_CTLPLANE_INTERFACE}
-edpm_baremetal: export OPERATOR_NAME=openstack
-edpm_baremetal: export EDPM_ANSIBLE_SECRET=${DATAPLANE_ANSIBLE_SECRET}
-edpm_baremetal: export EDPM_ANSIBLE_USER=${DATAPLANE_ANSIBLE_USER}
-edpm_baremetal: export BMAAS_NETWORK_NAME=${BM_NETWORK_NAME}
-edpm_baremetal: export BMAAS_NETWORK_IPADDRESS=${BM_NETWORK_IPADDRESS}
-edpm_baremetal: export BMAAS_INSTANCE_NAME_PREFIX=${BM_INSTANCE_NAME_PREFIX}
-edpm_baremetal: export EDPM_CHRONY_NTP_SERVER=${DATAPLANE_CHRONY_NTP_SERVER}
-edpm_baremetal: export EDPM_SSHD_ALLOWED_RANGES=${DATAPLANE_SSHD_ALLOWED_RANGES}
-edpm_baremetal: export EDPM_ROOT_PASSWORD_SECRET=${BM_ROOT_PASSWORD_SECRET}
-edpm_baremetal: export BMAAS_NODE_COUNT=${BM_NODE_COUNT}
-edpm_baremetal: export NODE_COUNT=${BM_NODE_COUNT}
-edpm_baremetal: export EDPM_TOTAL_NODES=${BM_NODE_COUNT}
-edpm_baremetal: export OPENSTACK_RUNNER_IMG=${DATAPLANE_CUSTOM_SERVICE_RUNNER_IMG}
-edpm_baremetal: export EDPM_REGISTRY_URL=${DATAPLANE_REGISTRY_URL}
-edpm_baremetal: export EDPM_CONTAINER_TAG=${DATAPLANE_CONTAINER_TAG}
+edpm_baremetal: export DATAPLANE_TOTAL_NODES=${BM_NODE_COUNT}
 edpm_baremetal: export EDPM_GROWVOLS_ARGS=${DATAPLANE_GROWVOLS_ARGS}
-edpm_baremetal: ## Deploy only dataplane with BMAAS
+edpm_baremetal: edpm_baremetal_bmaas
+	make -C .. edpm_deploy_baremetal
+
+.PHONY: edpm_baremetal_bmaas
+edpm_baremetal_bmaas: export OPERATOR_NAME=openstack
+edpm_baremetal_bmaas: export BMAAS_NETWORK_NAME=${BM_NETWORK_NAME}
+edpm_baremetal_bmaas: export BMAAS_INSTANCE_NAME_PREFIX=${BM_INSTANCE_NAME_PREFIX}
+edpm_baremetal_bmaas: export BMAAS_NODE_COUNT=${BM_NODE_COUNT}
+edpm_baremetal_bmaas: export NODE_COUNT=${BM_NODE_COUNT}
+edpm_baremetal_bmaas: export DEPLOY_DIR=../out/bmaas
+edpm_baremetal_bmaas: ## Create virtual baremetal for the dataplane
```

Comment:
I don't like this new target name, I'll rename it to edpm_baremetal_compute, to match the "edpm_compute" target